api/s2n.h

```diff
@@ -2870,6 +2870,19 @@ S2N_API extern int s2n_connection_client_cert_used(struct s2n_connection *conn);
  */
 S2N_API extern const char *s2n_connection_get_cipher(struct s2n_connection *conn);
 
+/**
+ * Provides access to the TLS-Exporter functionality.
+ *
+ * See https://datatracker.ietf.org/doc/html/rfc5705 and https://www.rfc-editor.org/rfc/rfc8446.
+ *
+ * @note This is currently only available with TLS 1.3 connections which have finished a handshake.
+ *
+ * @param conn A pointer to the connection
+ * @returns A POSIX error signal. If an error was returned, the value contained in `output` should be considered invalid.
+ */
+S2N_API extern int s2n_connection_tls_exporter(struct s2n_connection *conn, uint8_t *output, uint32_t output_length,
+        const uint8_t *label_, uint32_t label_length, const uint8_t *context, uint32_t context_length);
```

Comment:
s2n-tls doesn't have any naming conventions around trailing "\_". Are you using "\_" to indicate something?

Reply:
I created some blobs from the API parameters with the same name and C doesn't support aliasing, this was my workaround for that. Happy to try to come up with some other name for the blobs or the parameter though.

Reply:
Some name schemes I've seen elsewhere in the code:
label_in vs label
label_bytes vs label
label_data vs label
label vs label_blob